lib/Runtime/Library/JavascriptNumber.inl

```diff
@@ -148,7 +148,7 @@ namespace Js
     inline Var JavascriptNumber::ToVar(double value)
     {
         uint64 val = *(uint64*)&value;
-        AssertMsg(!IsNan(value) || ToSpecial(value) == k_Nan || ToSpecial(value) == 0x7FF8000000000000ull, "We should only produce a NaN with this value");
+//        AssertMsg(!IsNan(value) || ToSpecial(value) == k_Nan || ToSpecial(value) == 0x7FF8000000000000ull, "We should only produce a NaN with this value");
```

Comment:
>0x7FF8000000000000ull [](start = 86, length = 21)

Shouldn't this just be k_NegativeNan now?

Reply:
Yes